To: vim_dev@googlegroups.com
Subject: Patch 7.3.158
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.158
Problem:    Might use uninitialized memory in C indenting.
Solution:   Init arrays to empty.
Files:	    src/misc1.c


*** ../vim-7.3.157/src/misc1.c	2011-04-11 14:26:15.000000000 +0200
--- src/misc1.c	2011-04-11 14:11:17.000000000 +0200
***************
*** 6433,6438 ****
--- 6433,6440 ----
  	/* find how indented the line beginning the comment is */
  	getvcol(curwin, trypos, &col, NULL, NULL);
  	amount = col;
+ 	*lead_start = NUL;
+ 	*lead_middle = NUL;
  
  	p = curbuf->b_p_com;
  	while (*p != NUL)
*** ../vim-7.3.157/src/version.c	2011-04-11 14:26:15.000000000 +0200
--- src/version.c	2011-04-11 14:27:05.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     158,
  /**/

-- 
It's totally unfair to suggest - as many have - that engineers are socially
inept.  Engineers simply have different objectives when it comes to social
interaction.
				(Scott Adams - The Dilbert principle)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
